images/macos/toolsets/toolset-12.json

```diff
@@ -19,16 +19,30 @@
             "6.12.0.182"
         ],
         "ios-versions": [
-            "15.10.0.5", "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
+            "15.12.0.2", "15.10.0.5", "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
         ],
         "mac-versions": [
-            "8.10.0.5", "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
+            "8.12.0.2", "8.10.0.5", "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
         ],
         "android-versions": [
-            "12.2.8.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
+            "13.0.0.0", "12.3.3.3", "12.2.8.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
         ],
```

Comment:
These version numbers look right to me. How are the downloaded and installed on the image?

Reply:
We use script to install Xamarin - https://github.com/actions/runner-images/blob/main/images/macos/provision/core/xamarin.sh and helper functions - https://github.com/actions/runner-images/blob/main/images/macos/provision/utils/xamarin-utils.sh

Reply:
Ok, great.

What decides what version of Xamarin.Android this symlink points to?

    /Library/Frameworks/Xamarin.Android.framework/Versions/Current

Just whichever one was installed *last*?

Reply:
@jonathanpeppers , Do you think we should set up the last 6_12_19 bundle as the default?

#### Xamarin bundles
| symlink           | Xamarin.Mono | Xamarin.iOS | Xamarin.Mac | Xamarin.Android |
| ----------------- | ------------ | ----------- | ----------- | --------------- |
| 6_12_19           | 6.12         | 15.12       | 8.12        | 13.0            |
| 6_12_18 (default) | 6.12         | 15.10       | 8.10        | 12.3            |

Reply:
Yes, I would go ahead and make the latest the default every time you update these. Mobile platforms move so fast, we likely have fixes that are needed for latest Android SDK, NDK, Xcode, etc.

Reply:
@jonathanpeppers, Thank you, updated.